substrate/pkg/utils/discovery/tags.go

```diff
@@ -27,8 +27,12 @@ const (
 )
 
 func Tags(substrate *v1alpha1.Substrate, name *string) []*ec2.Tag {
+	return OwnerWithNameTag(aws.String(OwnerTagKey), aws.String(substrate.Name), name)
+}
+
+func OwnerWithNameTag(ownerTag, ownerValue, name *string) []*ec2.Tag {
```

Comment:
why this function ? unlike before ?

Reply:
Trying to remove dependency on *v1alpha1.Substrate in the client library using this function.